ansible/roles/uan_interfaces/tasks/customer_interfaces.yml

```diff
@@ -122,23 +110,22 @@
       timeout: 5
 
   - name: Remove existing default route
-    when: not uan_can_setup
+    when: uan_customer_default_route
     command: "ip route delete default"
     changed_when: result.rc != 0 or result.rc == 2
     failed_when: result.rc != 0 and result.rc != 2
     register: result
 
-  - name: ifup interfaces to set the interface config and routes
-    shell: "wicked ifup {{ item.name }}"
+  - name: Restart wicked to start interface config and routes
+    shell: "systemctl restart wicked"
     changed_when: result.rc == 0
     failed_when: result.rc != 0
     register: result
-    when: item.name is defined
-    loop: "{{ customer_uan_interfaces }}"
+    when: uan_customer_default_route or (customer_uan_interfaces | length)
 
-  - name: Wait 10 seconds for the interfaces to come up
+  - name: Wait 30 seconds for the interfaces to come up
     wait_for:
-      timeout: 10
+      timeout: 30
```

Comment:
Was this found to be needed during tested?

Reply:
After refactoring the PR, the longer timeout is not needed.